packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -214,18 +214,26 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
       privateArgs.assetInfo,
     );
   }
-
+  const nobleAccountV = zone.makeOnce('NobleAccount', () => makeNobleAccount());
   const feedKit = zone.makeOnce('Feed Kit', () => makeFeedKit());
 
   const poolAccountV = zone.makeOnce('PoolAccount', () => makeLocalAccount());
   const settleAccountV = zone.makeOnce('SettleAccount', () =>
     makeLocalAccount(),
   );
   // when() is OK here since this clearly resolves promptly.
-  /** @type {HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>[]} */
-  const [poolAccount, settlementAccount] = await vowTools.when(
-    vowTools.all([poolAccountV, settleAccountV]),
+  /** @type {[HostInterface<OrchestrationAccount<{chainId: 'noble-1';}>>, HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>, HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>]} */
+  const [nobleAccount, poolAccount, settlementAccount] = await vowTools.when(
+    vowTools.all([nobleAccountV, poolAccountV, settleAccountV]),
+  );
+  trace('settlementAccount', settlementAccount);
+  trace('poolAccount', poolAccount);
+  trace('nobleAccount', nobleAccount);
+
+  const intermediateRecipient = await vowTools.when(
```

Comment:
This address... it's not used for anything other than bech32 checksum validation, right?

How about using `bech32.encode('agoric', Array(20).fill(0))` aka `agoric1qqqqqqqqqqqqqqqqqqqqhlgkpj`?

Reply:
> This address... it's not used for anything other than bech32 checksum validation, right?

My answer to "How about using... ?" completely depends on the answer to the above (which I suspect *is* used for something other than checksum validation).  I don't know the answer to the above.